.yarnrc.yml

```diff
@@ -15,3 +15,6 @@ unsafeHttpWhitelist:
   - 127.0.0.1
 
 yarnPath: .yarn/releases/yarn-3.5.0.cjs
+checksumBehavior: "update"
```

Comment:
We do not want this in develop under any circumstance, but in the interim that will make CI happy

Reply:
95a9ca1e45305b7536352015a536c4083ac3bd68